app/assets/stylesheets/helpers.scss

```diff
@@ -26,15 +26,21 @@
   height: calc(100vh - 402px);
 }
 
+.cursor-pointer {
+  cursor: pointer;
+}
+
 .nav-tabs {
   border-bottom: 0;
+  padding-bottom: 1px;
 
   .nav-link {
     transition: none;
+    border: 0;
+
     &.active, &:hover {
       color: $primary;
-      border: 0;
-      border-bottom: 1px solid $primary;
+      border-bottom: 1px solid $primary !important;
     }
   }
```

Comment:
I think this should go in rooms.scss and be scoped under #rooms or something

Reply:
I can clean this up later

Reply:
we can put a div/wrapper with id="rooms" in component/rooms/Rooms.jsx